kubernetes/baseline-attack.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+# shellcheck disable=SC2001
+
+OUTPUT_DIR=$1
+
+/go_attack/kubernetes/log-git-commit.sh "$OUTPUT_DIR"
+
+python3 go_attack/scripts/baseline_attack.py \
+  --config go_attack/configs/gtp-base.cfg \
+  --models /shared/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz \
+    /shared/victims/kata1-b20c256x2-s5303129600-d1228401921.bin.gz \
+  --num-games 150 \
+  --policy edge mirror spiral random \
+  --num-visits 1 2 4 8 16 32 \
+  --passing-behavior standard avoid-pass-alive-territory \
+  --victim-color B W \
+  --moves-before-pass 800 \
+  --parallel-runs-per-gpu 12 \
+  --log-dir "$OUTPUT_DIR"/original
+
+# Rescore the results using KataGo scoring.
+mkdir -p "$OUTPUT_DIR/rescored"
+for DIR in "$OUTPUT_DIR/original/"*; do
+  MODEL=$(echo "$DIR" | sed "s/.*model=\([a-z0-9-]\+\).*/\1/")
+  VISITS=$(echo "$DIR" | sed "s/.*visits=\([0-9]\+\).*/\1/")
+  POLICY=$(echo "$DIR" | sed "s/.*policy=\([a-z]\+\).*/\1/")
+  VICTIM_COLOR=$(echo "$DIR" | sed "s/.*victim=\([A-Z]\).*/\1/")
+  PASSING_BEHAVIOR=$(echo "$DIR" | sed "s/.*pass=\([a-z\-]\+\).*/\1/")
+  if [ "$MODEL" = "kata1-b40c256-s11840935168-d2898845681" ]; then
```

Comment:
Consider defining a variable `CP505_MODEL` or similar that is used both here and in line 10 to avoid duplication and make this less magical?